src/components/bookclub/BookClub.jsx

```diff
@@ -0,0 +1,374 @@
+        address_line1: "123",
+        address_line2: "Front St W"
+      },
+      image_url: null
+    },
+    image:
+      "https://www.librarieshawaii.org/wp-content/uploads/2016/09/Book-club-graphic-resized-for-website.jpg",
+    users: [
+      {
+        id: 39,
+        name: "seed2",
+        email: "seed2@yahoo.com",
+        password_digest:
+          "$2a$12$WO.hphs0Pi8rFybLvR7hhe7I7t1nLWnpP8a5K6CNQCjF3eq.c3mei",
+        created_at: "2019-09-05T16:04:44.680Z",
+        updated_at: "2019-09-05T16:04:44.680Z",
+        address: {
+          city: "Toronto",
+          address_line1: "301",
+          address_line2: "Front St W"
+        }
+      },
+      {
+        id: 38,
+        name: "seed",
+        email: "seed@yahoo.com",
+        password_digest:
+          "$2a$12$GjjSSd.78s5lXK85R1QYEukjZ.pmUN4cRSs8bL8EHTDUAXq8I20km",
+        created_at: "2019-09-05T16:04:44.289Z",
+        updated_at: "2019-09-05T16:04:44.289Z",
+        address: {
+          city: "Toronto",
+          address_line1: "301",
+          address_line2: "Front St W"
+        }
+      }
+    ],
+    admins: [
+      {
+        id: 38,
+        name: "seed",
+        email: "seed@yahoo.com",
+        password_digest:
+          "$2a$12$GjjSSd.78s5lXK85R1QYEukjZ.pmUN4cRSs8bL8EHTDUAXq8I20km",
+        created_at: "2019-09-05T16:04:44.289Z",
+        updated_at: "2019-09-05T16:04:44.289Z",
+        address: {
+          city: "Toronto",
+          address_line1: "301",
+          address_line2: "Front St W"
+        }
+      }
+    ]
+  };
+    getBookClub(21).then(e=>e.json()).then(e=>{
+     console.log(e);
+
+     data = e
+   });
+
+  console.log("CLUB::", props);
+
+  // let users = null;
+  // if (data.users.length > 0) {
+  //   users = data.users.map((e, i) => <li key={i}>{e.name}</li>);
+  // }
+
+  return (
+    <Card className="text-center">
+      <Card.Header>Welcome to</Card.Header>
+      <Card.Body>
+        <Card.Title>{data.book_club.name}</Card.Title>
+        <Card.Text>
+          {data.book_club.description ||
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec facilisis feugiat "}
+        </Card.Text>
+
+        <div>
+          <div style={{ float: "left" }}>
+            <ListGroup variant="flush">
+              <ListGroup.Item>{data.userlist}</ListGroup.Item>
+            
+            </ListGroup>
+          </div>
+          <div style={{ float: "right" }}>
+            <ListGroup variant="flush">
+              <ListGroup.Item>{props.adminlist}</ListGroup.Item>
+              
+            </ListGroup>
+          </div>
+        </div>
+      </Card.Body>
+
+      <Button variant="primary">Go somewhere</Button>
+
+      <Card.Footer className="text-muted">
+        {`${data.book_club.address.address_line1} ${data.book_club.address.address_line2} ${data.book_club.address.city} `}
+      </Card.Footer>
+    </Card>
+  );
+};
+
+export default BookClubs;
+
+
+// import React, { Component } from "react";
```

Comment:
The commented code is more on target than the not commented code. It fetches the book club based on the book club id.